pkg/config/config.go

```diff
@@ -48,6 +48,10 @@ type Config struct {
 	Manifests Manifests     `json:"manifests"`
 	Ingress   IngressConfig `json:"ingress"`
 
+	// Settings specified in this section are transferred as-is into the Kubelet config.
+	// +kubebuilder:validation:Schemaless
+	Kubelet map[string]any `json:"kubelet"`
```

Comment:
`// +kubebuilder:default=null`
Given that you already have schemaless you can put whatever you want here, meaning `null` should work.